### ISSUE TITLE
Cardinal Components Expects ClientWorld Class in all Environments

### ISSUE THREAD

**Issue by amnotbananaama** [Ladysnake/Cardinal-Components-API#54]
Very minor bug, but Cardinal Components attempts to mixin into [`net.minecraft.client.world.ClientWorld`](https://github.com/OnyxStudios/Cardinal-Components-API/blob/40f27e90207a5109f59e6b13e47172b038b6bb91/cardinal-components-world/src/main/java/dev/onyxstudios/cca/mixin/world/client/MixinClientWorld.java#L26), even though that class does not exist on dedicated servers. 

This results in a `Error loading class: net/minecraft/class_638 (java.lang.ClassNotFoundException: net/minecraft/class_638)` printed to the server log. This has absolutely no impact on gameplay or the functionality of any mods, it just results in a bit of spam to server logs that would be great to fix.

Ideally, there would be some kind of check, and disable loading that mixin if the current environment is a dedicated server. Thanks for taking the time to read this, and for creating this API!

**Comment by UpcraftLP** [comment]
The mixin in question [is client-only already](https://github.com/OnyxStudios/Cardinal-Components-API/blob/40f27e90207a5109f59e6b13e47172b038b6bb91/cardinal-components-world/src/main/resources/mixins.cardinal_components_world.json#L11).

Could you maybe provide a modlist + full log?

**Comment by amnotbananaama** [comment]
Whoops! I'll try a binary search again, I thought I narrowed it down correctly

**Comment by UpcraftLP** [comment]
Well again, providing the actual log would be helpful, along with a list of all installed mods.

**Comment by amnotbananaama** [comment]
Reproduced it with just Cardinal-Components-API-2.6.0 on a 1.16.2 dedicated server.

https://pastebin.com/raw/UDgRh6Xn

```
[03:19:07] [main/WARN]: Error loading class: net/minecraft/class_638 (java.lang.ClassNotFoundException: net/minecraft/class_638)
[03:19:07] [main/WARN]: @Mixin target net.minecraft.class_638 was not found mixins.cardinal_components_block.json:common.MixinClientWorld
```

**Edit**: I think it's [this](https://github.com/OnyxStudios/Cardinal-Components-API/blob/40f27e90207a5109f59e6b13e47172b038b6bb91/cardinal-components-block/src/main/resources/mixins.cardinal_components_block.json#L10) then? Apologies for the delay in getting back to you on this

**Comment by UpcraftLP** [comment]
> Edit: I think it's this then? Apologies for the delay in getting back to you on this

Yes, that's it.

**Comment by Pyrofab** [comment]
Oh that's been fixed since 2.7.4. Thanks for the report !
